### PR TITLE
link mkl in gpu tao bridge for BlaDNN

### PR DESCRIPTION
**Pull request by minminsun**

### CODE REVIEW

File: tao_compiler/mlir/xla/ral/BUILD [hunk 1]
```diff
@@ -327,7 +327,13 @@ cc_library(
         "-fopenmp",
         "-ldl",
         "-lm"
-    ],
+    ] + if_blade_gemm([
+        "-Wl,--start-group",
+        "$(location //tao/third_party/mkldnn:build)/intel/lib/libmkl_intel_ilp64.a",
```
Comment (Orion34-lanbo) [inline tao_compiler/mlir/xla/ral/BUILD:332]: Can we use mkldnn download from bazel as `$(location @mkl_static//:lib)/libmkl_intel_ilp64.a` or so on.

File: tao_compiler/mlir/xla/ral/BUILD [hunk 2]
```diff
@@ -357,6 +363,10 @@ cc_library(
         ":disc_mkl_cmake",
         ":ral_md5",
     ]),
+    data = [
+    ] + if_blade_gemm([
```
Comment (Orion34-lanbo) [inline tao_compiler/mlir/xla/ral/BUILD:367]: Nit: delete the empty `[]`

File: tao_compiler/mlir/xla/ral/BUILD [hunk 3]
```diff
@@ -710,6 +726,11 @@ cc_library(
     ]) + if_mkldnn([
         ":disc_mkl_bazel",
         ":ral_md5",
+    ]) + [
+    ],
```
Comment (Orion34-lanbo) [inline tao_compiler/mlir/xla/ral/BUILD:730]: ditto

File: tao_compiler/mlir/xla/ral/BUILD (hunk 2 above)
Comment (minminsun) [reply tao_compiler/mlir/xla/ral/BUILD]: Done

File: tao_compiler/mlir/xla/ral/BUILD (hunk 3 above)
Comment (minminsun) [reply tao_compiler/mlir/xla/ral/BUILD]: Done

File: tao_compiler/mlir/xla/ral/BUILD (hunk 1 above)
Comment (minminsun) [reply tao_compiler/mlir/xla/ral/BUILD]: Done, and I've verified that it really works, as a result there's no need to download/build mkldnn in tao_build.py for gpu build. Thanks!

**Review by Orion34-lanbo — APPROVED** [review]
LGTM

**Pull request merged**